src/api-guides/travel/deeplink-integration-new.md

```diff
@@ -203,12 +203,54 @@ https://www.concursolutions.com/goto/hotel-shop?checkindate=2024-11-02&checkoutd
+|`location`      |`string` |[Location Format](/api-guides/travel/deeplink-integration.html#location-format-hotel)                                                              |**Required** <br>Location of the search center with default search radius of 5 mi.|
+|`locationsource`|`string` |one of {`northstar`, `giata`, `leonardo`, `amadeus`, `sabre`, `galileo`, `cwt`, `expedia`, `hrs`, `booking.com`, `omnibees`}                       |**Required if location is a Hotel Property ID**. <br>Source of the Hotel Property ID. For geo-coordinates and SAP Concur Travel Internal Hotel ID, omit this query parameter.|
+|`distance`      |`integer`|one of {`1`, `2`, `3`, `4`, `5`, `10`, `15`, `20`, `25`, `30`, `35`, `40`, `45`, `50`, `55`, `60`, `65`, `70`, `75`, `80`, `85`, `90`, `95`, `100`}|(Optional) <br> Distance around the `location`. <br>If not set, it will default to 5 mi `distance` and `radius` as search radius.|
+|`unit`          |`string` |`km` or `mi`                                                                                                                                       |(Optional) <br> Unit for the `distance`. <br>If not set, the value will be taken from users profile - under Profile Settings, System Settings, **mile/km**.|
+
+#### <a name="example-one-way"></a> Example: Hotel Property ID location
+```
+https://www.concursolutions.com/goto/hotel-shop?tripid=a7f172cd-8587-48d5-8533-5ddb7ff5c26c&location=708752&locationsource=hrs
+```
+
+This deeplink will perform a search for the hotel Travelodge LAX South in El Segundo (CA) and hotels nearby in the 5 mi default radius. It uses the Hotel Property ID from HRS as location. The check-in and check-out dates are taken from the corresponding trip with trip id `a7f172cd-8587-48d5-8533-5ddb7ff5c26c`. Either the trip contains air bookings, then the arrival date of the first flight will be used as check-in date for the hotel search and the departure date of the last flight will be used as check-out date, or the trip start and end dates will be used as check-in and check-out dates, respectively.
+
+#### <a name="example-one-way"></a> Example: Geo-coordinate location
+```
+https://www.concursolutions.com/goto/hotel-shop?tripid=a7f172cd-8587-48d5-8533-5ddb7ff5c26c&location=48.85694273527786,2.3501079080340315
+```
+
+This deeplink will pre-fill the hotel search form with Paris, in particular the coordinates `48.85694273527786,2.3501079080340315` (**todo: test how geo-coordinates are displayed in hotel search form**), as the search center with a 5 mi radius around it and with check-in and check-out dates taken from the trip corresponding to the trip id `a7f172cd-8587-48d5-8533-5ddb7ff5c26c`. Either the arrival date of the first flight will be used as check-in date and the departure of the last flight will be used as check-out date, or the trip start and end dates will be used, respectively.
```

Comment:
This will also directly start a search as a location is provided, therefore the coordinate display is the same as with the regular hotel deeplink (so not sure if it requires a specific mention).